python/hsc/pipe/s18a/undeblendedConvolved.py

```diff
@@ -0,0 +1,130 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
Do we need Python 2 support anymore?  I thought even HSC was on pure Python 3 now.

Also, while this is partially pre-existing, we should have some kind of copyright in this package and file.

Reply:
My notes for the 6.0-beta0 release say:
> * We default to installing python 3. If you want python 2, then supply a "-2" argument to newinstall.sh.

So I believe we are still supporting both for HSC.

I added a copyleft.